openpathsampling/analysis/tis_analysis.py

```diff
@@ -353,6 +343,9 @@ def total_crossing_probability(self, storage=None, method="wham", force=False):
         force : bool (False)
             if true, cached results are overwritten
         """
+
+        tcp = dict()
```

Comment:
The other disadvantage I remember is that autocomplete in ipynbks will not work until .tcp is created.
`analysis_object.<TAB>` will not list `tcp` as an option, which can be confusing.